### ISSUE TITLE
A bunch of GTK2 related bugs with missing assets

### ISSUE THREAD

**Issue by Ledarium** [godlyranchdressing/Minwaita#4]
Hello, it's me again. I had downloaded 1.3.2 and discovered some new bugs in GTK2.
![image](https://cloud.githubusercontent.com/assets/10389237/25586229/20144892-2ea7-11e7-96f8-fd6d73cc797b.png)
You can see missing radio button and ugly textbox borders near pushbutton. Log enclosed below. Hope it will be fixed soon! 
```
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:162: Unable to locate image file in pixmap_path: "assets/menu-checkbox-active.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:164: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:202: Unable to locate image file in pixmap_path: "assets/menu-checkbox-checked-active.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:204: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:242: Unable to locate image file in pixmap_path: "assets/menu-checkbox-mixed-active.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:244: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:270: Unable to locate image file in pixmap_path: "assets/radio.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:272: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:278: Unable to locate image file in pixmap_path: "assets/radio-hover.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:280: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:286: Unable to locate image file in pixmap_path: "assets/radio-active.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:288: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:294: Unable to locate image file in pixmap_path: "assets/radio.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:296: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:302: Unable to locate image file in pixmap_path: "assets/radio-insensitive.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:304: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:310: Unable to locate image file in pixmap_path: "assets/radio-checked.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:312: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:318: Unable to locate image file in pixmap_path: "assets/radio-checked-hover.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:320: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:326: Unable to locate image file in pixmap_path: "assets/radio-checked-active.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:328: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:334: Unable to locate image file in pixmap_path: "assets/radio-checked.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:336: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:342: Unable to locate image file in pixmap_path: "assets/radio-checked-insensitive.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:344: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:350: Unable to locate image file in pixmap_path: "assets/radio-mixed.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:352: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:358: Unable to locate image file in pixmap_path: "assets/radio-mixed-hover.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:360: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:366: Unable to locate image file in pixmap_path: "assets/radio-mixed-active.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:368: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:374: Unable to locate image file in pixmap_path: "assets/radio-mixed.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:376: Overlay image options specified without filename
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:382: Unable to locate image file in pixmap_path: "assets/radio-mixed-insensitive.png"
/usr/share/themes/Minwaita-Dark-Vanilla/gtk-2.0/main.rc:384: Overlay image options specified without filename
```

**Comment by Ledarium** [comment]
I had fixed it, take a look at pull request. #5 (But ugly textbox is still present, also in light theme)

**Comment by godlyranchdressing** [comment]
Whoops. I really appreciate your effort but I'm afraid I worked ahead of you. 

The problem falls back to the "-active" assets not being  in the assets.svg file. Though they _are_ included in the pre-installed GTK2 theme for Adwaita. I just went ahead and did some up using the GTK3 checkboxes and radio boxes as the base as well as just make the other states for the GTK2 checkboxes and radio buttons match their GTK3 counterparts. I don't know why they don't. The hover state for both elements look especially ugly in the GTK2 theme, appearing as a hollow box and circle. I'll be pushing it soon.
